Replace instructions to modify pubspec.yaml with `flutter pub add` command in multiple pages (#8950)
flutter/website: src/cookbook/persistence/key-value.md

## 1. Add the dependency

> Before starting, add the [`shared_preferences`][] dependecy to your project.

Reviewer: ```suggestion
Before starting, add the [`shared_preferences`][] package as a dependency.
```
